target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -312,6 +312,13 @@ whr-600d)
 	ucidef_set_led_default "power" "power" "$board:green:power" "1"
 	ucidef_set_led_default "router" "router" "$board:green:router" "1"
 	;;
+wl-wn575a3)
+	ucidef_set_rssimon "wlan1" "40000" "1"
+	ucidef_set_led_rssi "wifi-low" "wifi-low" "$board:green:wifi-low" "wlan1" "1" "49" "0" "1"
+	ucidef_set_led_rssi "wifi-med" "wifi-med" "$board:green:wifi-med" "wlan1" "50" "84" "0" "1"
+	ucidef_set_led_rssi "wifi-high" "wifi-high" "$board:green:wifi-high" "wlan1" "85" "100" "0" "1"
+	set_wifi_led "$board:green:wps" "wlan1"
```

Comment:
Why are you using the WPS led for wlan1? Doesn't work the rssi stuff for you or do I miss here something?